doc/src/build/json-rpc.md

```diff
@@ -27,38 +27,28 @@ Use the following command to start an RPC server:
 ```shell
 $ rpc-server
```

Comment:
`rpc-server` is equivalent to Gateway. we should remove relevant sections in this page and link to `fullnode.md` for how to run a fullnode

Reply:
updated